packages/faustwp-cli/utils/getGraphqlEndpoint.ts

```diff
@@ -0,0 +1,10 @@
+import { getWpUrl } from './getWpUrl.js';
+
+/**
+ * Returns the GraphQL Endpoint.
+ */
+export function getGraphqlEndpoint() {
+  const wpUrl = getWpUrl();
+
+  return `${wpUrl}/graphql`;
```

Comment:
This could be problematic if the users change that using the `https://www.wpgraphql.com/filters/graphql_endpoint` filter

Reply:
Thanks @theodesp, I absolutely agree. [See my comment here](https://github.com/wpengine/faustjs/pull/1318#discussion_r1132522872).